DatabaseAndReporting/Integration/02_ManageQueryCategories.md

```diff
@@ -0,0 +1,218 @@
+
+[comment]: <> (Key wird hier irgendwann abgebaut)
+
+6. Click the [SAVE] button.   
+    The query sub-category has been created. The *Add element* window is closed. The new query sub-category is displayed in the query sub-category column.
+
+    ![Query sub-category](../../Assets/Screenshots/DatabaseAndReporting/Settings/QueryCategories/QuerySubcategoryCreated.png "[Query sub-category]")
+
+    > [Info] If you want to change the order of the query sub-categories in the list, see [Change the order of query categories](#change-the-order-of-query-categories).
+
+7. Click the [SAVE] button in the upper right corner.   
+    The *Submitting data...* view is displayed shortly while saving. 
+
+    ![Submitting data](../../Assets/Screenshots/DatabaseAndReporting/Settings/QueryCategories/SubmittingData.png "[Submitting data]")
+
+    The new query sub-category has been saved to the query categories tree, which is displayed in the *Queries* view.
+
+    > [Info] It may be necessary to press the **F5** key to initialize the *Core1 Platform* for the changes to be displayed in the *Queries* view. 
+
+
+### Edit the query category data
+
+Edit the query category name and key as necessary.
+
+The procedure to edit the data of query categories and sub-categories is identical. Just select a sub-category instead of a category and follow the steps described below.
+
+#### Prerequisites 
+
+At least one query category has been created, see [Create a query category](#create-a-query-category).
+
+#### Procedure
+
+*Database and reporting > Settings > Tab QUERY CATEGORIES*
+
+![Query categories](../../Assets/Screenshots/DatabaseAndReporting/Settings/QueryCategories/QueryCategory.png "[Query categories]")
+
+1. Click the ![Edit](../../Assets/Icons/Edit03.png "[Edit]") (Edit) button to the right of the query category name.   
+    The query category data is displayed in a window in the query category column. 
+
+    ![Query categories](../../Assets/Screenshots/DatabaseAndReporting/Settings/QueryCategories/EditQueryCategoryData.png "[Query categories]")
+
+2. Modify the query category data as necessary.
+
+3. Click the [SAVE] button in the bottom right corner of the query category data window.  
+    The changes have been saved to the query category column. 
+
+    > [Info] If you want to change the order of the query sub-categories in the list, see [Change the order of query categories](#change-the-order-of-query-categories).
+
+4. Click the [SAVE] button in the upper right corner.  
+     The *Submitting data...* view is displayed shortly while saving. 
+
+    ![Submitting data](../../Assets/Screenshots/DatabaseAndReporting/Settings/QueryCategories/SubmittingData.png "[Submitting data]")
+
+    The changes have been saved to the query categories tree, which is displayed in the *Queries* view.
+
+    > [Info] It may be necessary to press the **F5** key to initialize the *Core1 Platform* for the changes to be displayed in the *Queries* view. 
```

Comment:
```suggestion
    > [Info] It may be necessary to press the **F5** key to initialize the *Core1 Platform* and to display the changes in the *Queries* view. 
```